challenges/KICKin-Rollin-Challenge.tex

```diff
@@ -0,0 +1,46 @@
+% !TeX root = ../SPL-Challenges.tex
+% !TeX spellcheck = en_US
+\section{KICKin\textquotesingle \& Rollin\textquotesingle Challenge}
+
+The objective of this challenge is for a robot to successfully kick a rolling ball from a ramp into a designated goal area. 
+Each team participating will record three attempts spread accross three days of the competition.
+An attempt consist of three kick attempts. 
+
+\subsection{Field Setup}
+
+The challenge will take place on a standard SPL field with its official marking. 
+The ramp will be positioned perpendicularly adjacent to the halfway line, aligned with a virtual line extending from the extremity of the center circle on the opponent's side.
```

Comment:
What does perpendicularly adjacent to the halfway line mean? Perpendicular to what? Which direction will the ball roll?